tests/test_models.py

```diff
@@ -2655,6 +2655,21 @@ def test_set_privacy(self):
         assert_false(self.project.is_public)
         assert_equal(self.project.logs[-1].action, NodeLog.MADE_PRIVATE)
 
+    @mock.patch('website.project.model.mails.queue_mail')
+    def test_set_privacy_sends_mail(self, mock_queue):
+        self.project.set_privacy('private', self.consolidate_auth)
+        self.project.set_privacy('public', self.consolidate_auth)
+        assert_true(mock_queue.called_once())
+        self.project.set_privacy('private', self.consolidate_auth)
+        self.project.set_privacy('public', self.consolidate_auth, skip_mail=False)
+        assert_true(mock_queue.called_once())
```

Comment:
For these simple checks with mock I would recommend using `assert_true(mock_queue.called)` or `assert_false(mock_queue.called)`